hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpGetCommandProcessor.java

```diff
@@ -78,7 +79,12 @@ private void handleMap(HttpGetCommand command, String uri) {
         int indexEnd = uri.indexOf('/', URI_MAPS.length());
         String mapName = uri.substring(URI_MAPS.length(), indexEnd);
         String key = uri.substring(indexEnd + 1);
-        Object value = textCommandService.get(mapName, key);
+        Object value;
+        if(MAP_ENTRIES_COMMAND.equalsIgnoreCase(key)){
```

Comment:
An additional REST command should not break any compatibility.
@asimarslan @sancar WDYT?